### DISCUSSION TITLE
problems loading ESP32 firmware from a BTT SKR 3 board.

### DISCUSSION THREAD

**Discussion by dstrohl** [luc-github/ESP3D#792] [Questions]
I seem to be unable to get the ESP32 wifi board to install on the system.

My steps:
I installed a ESP32 board on a BTT SKR 3 EZ motherboard per the docs,
I configured Marlin serial port 3 per the docs and installed that version of marlin firmware on the SKR board.
I downloaded the current ESP firmware from the wiki link and compiled it in Platform IO. *no changes in any config files.
I copied the firmware.bin file from the esp8266 directory to an sd card and named it esp3d.bin, then inserted that in the board's SD card slot and reset the board. (the rename was per the SKR docs which said to do that).
the red status lite blinked for a few minutes on both the SKR and ESP boards... then the status lite went out.
according to the docs, the file should be renamed to esp3d.cur afterwards, but the .bin file remained.

At this point. I am not seeing a wifi network with the name of ESP3D. I AM seeing one called ESP_xxxxxx (with the xxxxxx being the last 6 of the MAC address on the board) when I connect to that network, (no password), it provides an address in the 192.168.4.x range, and the .1 host does not accept any connections. (since this popped up immediately after the board was brought up, and goes away when it's powered down, and since the xxxx matches the MAC address, I'm pretty sure it's being generated by the card.  but I don't know what it is since that's not mentioned in the docs anywhere.

The instructions seem to imply that I need to install the firmware before installing the card on the board...  but all of the examples that I've found seem to be pointing to a different hardware.

**Comment by luc-github** [comment]
you write you have and esp32 board and you use esp8266 fw, this is not right what is your board ?
you write you connected `the ESP32 board on a BTT SKR 3 EZ motherboard per the docs` can you be more clear? like this there is no way to know you did it right and according my first question - I have strong doubts
Also which link are you refering to ? what is the version you are using ?
you wrote you compiled using platformIO but you did not mentionned what env you used to compile.

It is very hard to answer with such level of information, please complete the description or open an issue ticket and follow the template

**Reply by dstrohl** [reply]
I apologize for not being clearer... I will open an issue-ticket.
